Features5/tests/editContact.feature

```diff
@@ -0,0 +1,29 @@
+Feature: edit contact
```

Comment:
For this edit feature make a very simple feature file without any complex:
like 
make scenarios 1 as : make or add a single user and edit its data
make scenarios 2 as: make or add multiple user to contact and edit a specific user data by providing name or email

these two would be enough